addons/hr/tests/test_hr_employee.py

```diff
@@ -608,6 +608,39 @@ def test_user_creation_from_employee_multi_emails(self):
         self.assertEqual(params.get('message'), f"Users {employees[1].name} creation successful")
         self.assertTrue(employees[1].user_id)
 
+    def test_version_cron_update_no_phone_override(self):
+        """
+        Employees that already have a phone number should not be overridden by
+        the phone number of their company.
+        """
```

Comment:
Small nitpick but this docstring is a bit confusing grammatically and it might not be super clear to random people what it's actually saying. What about something like this? 
```suggestion
        """
        Employees with existing work phone number should keep it and not have 
        their number changed to the company phone number during the CRON.
        """
```

Reply:
Excellent idea. You're a poet that happens to be a programmer